zzzz_modular_occulus/code/modules/mob/living/simple_animal/hostile/siren/turret.dm

```diff
@@ -0,0 +1,166 @@
+	////Siren Turret////
+
+#define TURRET_STATE_NEUTRAL 0
+#define TURRET_STATE_AGGRO 1
+#define TURRET_STATE_WARMUP 2
+#define TURRET_STATE_ACTIVE 3
+#define TURRET_STATE_RECOVERY 4
+
+/mob/living/simple_animal/hostile/siren/turret
+	name = "monitor"
+	desc = "An innocious mellatic pod that floats in place. It seems to output a great deal of energy."
+	maxHealth = 100
+	health = 100
+	melee_damage_lower = 50
+	melee_damage_upper = 50
+	icon = 'zzzz_modular_occulus/icons/mob/siren/general.dmi'
+	icon_state = "passive"
+	icon_living = "passive"
+	minimum_distance = 0
+	move_to_delay = 60
+	vision_range = 12
+	aggro_vision_range = 15
+	ranged = TRUE
+	ranged_cooldown_time = 10
+	projectiletype = /obj/item/projectile/beam/sniper/siren
+	projectilesound = 'zzzz_modular_occulus/sound/effects/beamfire.ogg'
+	anchored = TRUE
+	var/combatant_state = TURRET_STATE_NEUTRAL
+	var/mob/living/beam_debuff_target
+	var/targetting = 0
```

Comment:
`targetting = FALSE`
(also, it'd be _targeting_, but that's not important - there's plenty of awkward spellings throughout the code.)